mod-source-record-storage-server/src/test/java/org/folio/dao/RecordDaoImplTest.java

```diff
@@ -138,6 +138,20 @@ public void shouldReturnIdOnStreamMarcRecordIdsWhenThereIsNoTrackingRecordAndFal
     });
   }
 
+  @Test
+  public void shouldReturnFalseWhenPreviousIndexersDeletionIsInProgress(TestContext context) {
+    Async async = context.async();
+
+    recordDao.deleteMarcIndexersOldVersions(TENANT_ID);
```

Comment:
Is there any guarantee that this call would still be running when the next call happens? Thinking it might get flaky.

Reply:
there is no strict and explicit mechanism in the test that guarantees it. This behavior is achieved due to the relative "heaviness" of the operations to the DB, which are performed in this method, so, suppose there is a probability of flaky test behavior. Changed the test to avoid the probability of the test getting flaky